### ISSUE TITLE
GMOD 64-Bit Branch Support?

### ISSUE THREAD

**Issue by sambow23** [BlueAmulet/SourceRTXTweaks#9]
Remix works on the 64-bit branch of Garry's Mod but it runs into the same issues as unpatched 32-bit. The patcher is unable to patch the 64-bit dlls though. Is it possible to introduce support for the 64-bit branch? 
![image](https://github.com/BlueAmulet/SourceRTXTweaks/assets/14916663/86feac07-3e8b-487d-9c9d-9f9fc079827d)

**Comment by BlueAmulet** [comment]
It's long been possible to run remix on the 64bit branch, I just never investigated it since people have been replacing various source engine dlls with ones from portal rtx which is a 32bit game. Are those dlls becoming unnecessary?

**Comment by BlueAmulet** [comment]
Some patches have been added for the 64bit version of GMod, but the code is a bit hard to follow and seems like more inlining has occurred. Various bits of frustum culling still occurs so I will keep this issue open for now

**Comment by cast-e** [comment]
Is there any progress in this issue? I would really want to run Garry's Mod in 64bits with RTX Remix but i don't know if it will work as well as the 32bit version.

**Comment by sambow23** [comment]
Gonna close this since x64 works good enough for now :3
